client/my-sites/plugins/plugin-information/index.jsx

```diff
@@ -147,6 +148,19 @@ export default React.createClass( {
 		return {};
 	},
 
+	getDefaultActionLinks( plugin ) {
+		let adminUrl = get( this.props, 'selectedSite.options.admin_url' );
```

Comment:
Oh and this breaks for me with Vaultpress, since the `get` falls back to `undefined` :slightly_frowning_face: (I'm pointed to `undefined/admin.php?page=vaultpress`)